### PR TITLE
Add health check for post install

### PR DESCRIPTION
**Pull request by yunjiang29**

### CODE REVIEW

**Comment by yunjiang29** [comment]
/ph-rehearse periodic-ci-openshift-openshift-tests-private-release-4.13-amd64-nightly-aws-usgov-ipi-disconnected-private-p2-f14

File: ci-operator/step-registry/cucushift/installer/check/cucushift-installer-check-commands.sh [hunk 1]
```diff
@@ -1,7 +1,6 @@
 #!/bin/bash
 
 set -o nounset
-set -o errexit
 set -o pipefail
```
Comment (jianlinliu) [inline ci-operator/step-registry/cucushift/installer/check/cucushift-installer-check-commands.sh:4]: Let us delete this line too.

File: ci-operator/step-registry/cucushift/installer/check/post-install/cucushift-installer-check-post-install-chain.yaml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+chain:
+  as: cucushift-installer-check-post-install
```
Comment (jianlinliu) [inline ci-operator/step-registry/cucushift/installer/check/post-install/cucushift-installer-check-post-install-chain.yaml:2]: can we directly name it `cucushift-installer-check`?

**Comment by yunjiang29** [comment]
/ph-rehearse periodic-ci-openshift-openshift-tests-private-release-4.13-amd64-nightly-aws-usgov-ipi-disconnected-private-p2-f14

**Comment by yunjiang29** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.12-arm64-nightly-aws-ipi-disconnected-private-tp-p3-f28

File: ci-operator/step-registry/cucushift/installer/check/cucushift-installer-check-commands.sh [hunk 3]
```diff
@@ -1,8 +1,8 @@
 #!/bin/bash
 
 set -o nounset
-set -o errexit
-set -o pipefail
+# set -o errexit
```
Comment (jianlinliu) [inline ci-operator/step-registry/cucushift/installer/check/cucushift-installer-check-commands.sh:4]: Let us copy the step to `cucushift-installer-check-print-info`, and include it in `cucushift-installer-check` chain.

Comment (yunjiang29) [reply ci-operator/step-registry/cucushift/installer/check/cucushift-installer-check-commands.sh:4]: remove it from test phase?

**Comment by yunjiang29** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.12-arm64-nightly-aws-ipi-disconnected-private-tp-p3-f28

**Comment by yunjiang29** [comment]
Result: [cucushift-installer-check-cluster-health](https://gcsweb-qe-private-deck-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/qe-private-deck/pr-logs/pull/openshift_release/40760/rehearse-40760-periodic-ci-openshift-openshift-tests-private-release-4.12-arm64-nightly-aws-ipi-disconnected-private-tp-p3-f28/1674331298405027840/artifacts/aws-ipi-disconnected-private-tp-p3-f28/cucushift-installer-check-cluster-health/)
[cucushift-installer-check-info](https://gcsweb-qe-private-deck-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/qe-private-deck/pr-logs/pull/openshift_release/40760/rehearse-40760-periodic-ci-openshift-openshift-tests-private-release-4.12-arm64-nightly-aws-ipi-disconnected-private-tp-p3-f28/1674331298405027840/artifacts/aws-ipi-disconnected-private-tp-p3-f28/cucushift-installer-check-info/)

Work as expected.

**Comment by yunjiang29** [comment]
@gpei @jianlinliu please take a look, ready for merge now

**Pull request merged**